server/paymentProviders/utils.ts

```diff
@@ -291,3 +293,6 @@ export const getOrCreateVendor = async (vendorProviderId, vendorName) => {
 
   return vendor;
 };
+
+export const hashObject = (obj: object) =>
+  crypto.createHash('sha1').update(JSON.stringify(obj)).digest('hex').slice(0, 7);
```

Comment:
@kewitz what prevents us from using a different hashing algorithm?

Reply:
Just the existing hashes. We could write a migration to recalculate those hashes.

Reply:
Ok! It's fine to do it later on, then.